cluster-image-builder/serve/vllm/v1/app.py

```diff
@@ -122,10 +122,20 @@ def __init__(self,
         self.openai_serving_embedding = None
         self.openai_serving_score = None
         self.openai_serving_models = None
-
+        
+        ctx = _serve.get_replica_context()
```

Comment:
Should it be calling `serve.get_replica_context()` ？

Reply:
my bad, fixed